packages/application/src/requests.tsx

```diff
@@ -25,40 +24,6 @@ import * as React from 'react';
  * A service class for making requests to the jupyter lab server.
  */
 export class RequestHandler {
```

Comment:
This file should also be renamed to .ts and you will most likely see that the inline React notation for HTML will break. I also noticed we are returning showDialog on 404 handler which should not be on this package if we are removing UI pieces altogether.